### PR TITLE
Fix Required PR: for pulp-smash

### PR DESCRIPTION
**Pull request by dralley**
[noissue]

### CODE REVIEW

File: templates/github/.github/workflows/scripts/before_install.sh.j2 [hunk 1]
```diff
@@ -87,22 +87,21 @@ cd ..
...
+  cd pulp-smash
+  git fetch --depth=1 origin pull/$PULP_SMASH_PR_NUMBER/head:$PULP_SMASH_PR_NUMBER
+  git checkout $PULP_SMASH_PR_NUMBER
+  cd ..
+  pip install --upgrade --force-reinstall ./pulp-smash
+fi
+
 # When building a (release) tag, we don't need the development modules for the
 # build (they will be installed as dependencies of the plugin).
 if [ "${TAG_BUILD}" = "1" ]; then
```
Comment (fao89) [inline templates/github/.github/workflows/scripts/before_install.sh.j2:102]: I think you can remove this if

Comment (dralley) [reply templates/github/.github/workflows/scripts/before_install.sh.j2:102]: The entire thing?

Comment (fao89) [reply templates/github/.github/workflows/scripts/before_install.sh.j2:102]: yes, from what I see, the if is just repeating:
```python
pip install --upgrade --force-reinstall ./pulp-smash
```

File: templates/github/.github/workflows/scripts/before_install.sh.j2 [hunk 2]
```diff
@@ -87,22 +87,21 @@ cd ..
...
 
 {% if plugin_name != 'pulpcore' %}
+git clone --depth=1 https://github.com/pulp/pulp-smash.git
+
+if [ -n "$PULP_SMASH_PR_NUMBER" ]; then
+  cd pulp-smash
+  git fetch --depth=1 origin pull/$PULP_SMASH_PR_NUMBER/head:$PULP_SMASH_PR_NUMBER
+  git checkout $PULP_SMASH_PR_NUMBER
+  cd ..
+  pip install --upgrade --force-reinstall ./pulp-smash
```
Comment (fao89) [inline templates/github/.github/workflows/scripts/before_install.sh.j2:97]: and this would be outside of this if

**Review by fao89 — APPROVED** [review]
Thank you!

**Pull request merged**